src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -2613,6 +2613,33 @@ def Stop() -> None:
     """
     pass
 
+def OnStop(callback: "Any" = None) -> None:
```

Ruff (0.15.20)

[error] Undefined name `Any`

(F821)